medallion/backends/memory_backend.py

```diff
@@ -104,13 +107,19 @@ def _pop_expired_sessions(self):
             self.next.pop(item)
 
     def load_data_from_file(self, filename):
-        with open(filename, "r") as infile:
-            self.data = json.load(infile)
+        if isinstance(filename, string_types):
+            with io.open(filename, "r", encoding="utf-8") as infile:
+                self.data = json.load(infile)
+        else:
+            self.data = json.load(filename)
 
     def save_data_to_file(self, filename, **kwargs):
         """The kwargs are passed to ``json.dump()`` if provided."""
-        with open(filename, "w") as outfile:
-            json.dump(self.data, outfile, **kwargs)
+        if isinstance(filename, string_types):
+            with io.open(filename, "w", encoding="utf-8") as outfile:
```

Comment:
could we place the encoding in the function itself and pass it to io.open(). In the event someone wishes to use a different one?